### PR TITLE
added waiting for bucket to exist.

### PR DESCRIPTION
**Pull request by dknochen**
fixing these errors:

image

### CODE REVIEW

**Comment by dknochen** [comment]
@wcandillon ready for review

File: tasks/s3.js [hunk 1]
```diff
@@ -96,11 +112,17 @@ var createBucket = function() {
         ACL : 'public-read'
     }, function(err, data) {
         if (err || data === null) {
-            $.util.log(bucketName + err);
+            $.util.log($.util.colors.red(bucketName + err));
             defered.reject();
         } else {
             $.util.log('createBucket(' + bucketName + ')');
-            defered.resolve();
+            waitForBucketExists()
```
Comment (wcandillon) [inline tasks/s3.js]: `return waitForBucketExists()`

**Comment by wcandillon** [comment]
@dknochen Looks good but this instabilities appears since we switched to docker. I'm wondering if this is linked. Also each build now creates a lot of bucket we need to clean up maybe.

**Comment by dknochen** [comment]
@wcandillon whenever it worked so far it was pure luck because the createBucket is asynchronous. So, it is a race condition in any case. With docker it became faster and the race condition started to show as failure.

**Pull request merged**